cmd/server/main.go

```diff
@@ -1,25 +1,33 @@
 package main
 
 import (
+	"database/sql"
 	"fmt"
 	"net/http"
 
 	"github.com/src-d/gitbase-playground/server"
 	"github.com/src-d/gitbase-playground/server/handler"
 	"github.com/src-d/gitbase-playground/server/service"
 
+	_ "github.com/go-sql-driver/mysql"
 	"github.com/kelseyhightower/envconfig"
 )
 
 // version will be replaced automatically by the CI build.
 // See https://github.com/src-d/ci/blob/v1/Makefile.main#L56
 var version = "dev"
 
+// Note: maxAllowedPacket must be explicitly set for go-sql-driver/mysql v1.3.
+// Otherwise gitbase will be asked for the max_allowed_packet column and the
+// query will fail.
+// The next release should make this parameter optional for us:
+// https://github.com/go-sql-driver/mysql/pull/680
 type appConfig struct {
 	Env       string `envconfig:"ENV" default:"production"`
 	Host      string `envconfig:"HOST" default:"0.0.0.0"`
```

Comment:
Sorry if I missed that somehow, not very related to the change, but while we are here - did not we have issues with this before, being not a valid IP on some OSes and decided that default should be localhost or 127.0.0.1 instead?

Remember discussing it with @smacker

Reply:
is not `0.0.0.0` the right one?
That's how it was configured in CAT, because that way the container is listening "everywhere"
https://github.com/src-d/code-annotation/pull/153/files

Reply:
`0.0.0.0` is a correct one. We use this variable for the server. And server should listen on all IPs.